src/goto-symex/symex_assign.cpp

```diff
@@ -227,11 +227,117 @@ void goto_symext::symex_assign_symbol(
     tmp_ssa_rhs.swap(ssa_rhs);
   }
 
+  state.field_sensitivity(ns, ssa_rhs, false);
```

Comment:
Unclear whether this does anything or just accrues state -- give the function name a verb